### DISCUSSION TITLE
[Esame 9 Settembre 2019] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#227] [Esami - Bertini @ Informatica]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/8264dbd2-246e-4ca7-9e50-c047e5ae8f36)

**Comment by LeoCosta02** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/46d72bc3-e902-49bb-9e27-4cb0dc7c9b80)

**Comment by Elia-Belli** [comment]
Sia $S$ la v.a. somma di $R,B$ rispettivamente il risultato del dado rosso e blu

**(1)** Se il dado rosso fa $5$ per ottenere $10$ il dado blu deve fare $5$ o $6$:

$$\mathbb{P}(S\geq 10| R=5)=\mathbb{P}(B=5|R=5)+\mathbb{P}(B=6|R=5)=\frac{1}{6}+\frac{1}{6}=\frac{1}{3}$$

**(3)** 

$$\mathbb{P}(R=5|S\geq 10)=\mathbb{P}(R=5| S=10)+\mathbb{P}(R=5|S=11)=\mathbb{P}(R=5|B=5)+\mathbb{P}(R=5|B=6)=\frac{1}{6}+\frac{1}{6}=\frac{1}{3}$$

**Comment by LeoCosta02** [comment]
@Elia-Belli al punto uno e due ho usato la formula del numero dei casi possibili fratto quello dei casi totali,per il punto 1 il risultato é uguale al tuo , mentre per il punto due il risultato mi viene 3/11

**Comment by cornflexxx** [comment]
$2)$ Ci sono tre casi  in cui in uno dei due dadi esce 5 e la loro somma è $\ge 10$, ovvero $(5,6),(6,5),(5,5)$, mentre i casi possibili in cui uno dei due è $5$ sono 11, ovvero $(5,i)$ e $(i,5)$ con $i=1,...,6$ , quindi  $\mathbb{P}(S\ge 10|R=5 \cup B=5)=3/11$.

**Comment by LeoCosta02** [comment]
@cornflexxx consideri 2 volte 5B e 5R? Io avevo pensato che erano 11 considerando una volta sola 5B e 5R
